vmdb/app/models/container_node.rb

```diff
@@ -8,4 +8,10 @@ class ContainerNode < ActiveRecord::Base
   belongs_to :lives_on, :polymorphic => true
 
   delegate   :hardware, :to => :computer_system
+
+  virtual_column :condition_ready, :type => :string, :uses => :container_node_conditions
+
+  def condition_ready
+    container_node_conditions.find_by_name('Ready').status
```

Comment:
@dkorn The method name `condition_ready` is confusing as I would think its contents would be `container_node_conditions.find_by_name('Ready')`.  So, I think that this method should be called something like `ready_condition_status`.  Suggest something like the following:

```ruby
def ready_condition
  container_node_conditions.find_by_name('Ready')
end

def ready_condition_status
  c = ready_condition
  return 'None' if c.nil?
  c.status
end
```